src/pages/index.tsx

```diff
@@ -558,241 +333,296 @@ const HomePage = ({
+            <SectionHeading>
               <Translation id="page-index-developers" />
-            </StyledH2>
-            <Subtitle>
+            </SectionHeading>
+            <SectionDecription>
               <Translation id="page-index-developers-description" />
-            </Subtitle>
-            <ButtonRow>
-              <ButtonLink to="/developers/">
-                <Translation id="page-index-developers-button" />
-              </ButtonLink>
-            </ButtonRow>
-          </LeftColumnContent>
-        </FeatureContent>
-        <CodeboxModal
-          isOpen={isModalOpen}
-          setIsOpen={setModalOpen}
-          title={codeExamples[activeCode].title}
-        >
-          <Codeblock
-            codeLanguage={codeExamples[activeCode].codeLanguage}
-            allowCollapse={false}
-            fromHomepage
+            </SectionDecription>
+            <ButtonLinkRow
+              firstButton={{
+                to: "/dapps/?category=technology",
+                child: <Translation id="page-index-developers-button" />,
+              }}
+            />
+          </FeatureContent>
+          <StyledCodeModal
+            isOpen={isModalOpen}
+            setIsOpen={setModalOpen}
+            title={codeExamples[activeCode].title}
+            sx={{
+              ".modal-component-container": {
+                padding: 0,
+                left: 0,
+                right: 0,
+                bottom: 0,
+                top: "50%",
+              },
+              ".modal-component": {
+                maxWidth: "100%",
+                maxHeight: "50%",
+                padding: 0,
+              },
+              ".modal-component-content": {
+                marginTop: "3rem",
+                width: "100%",
+                overflow: "auto",
+              },
+            }}
           >
-            {codeExamples[activeCode].code}
-          </Codeblock>
-        </CodeboxModal>
-      </DeveloperContainer>
-      <StyledGrayContainer>
-        <StyledContent>
+            <Codeblock
+              codeLanguage={codeExamples[activeCode].codeLanguage}
+              allowCollapse={false}
+              fromHomepage
+            >
+              {codeExamples[activeCode].code}
+            </Codeblock>
+          </StyledCodeModal>
+        </Row>
+      </MainSectionContainer>
+      {/* Eth Today Section */}
+      <GrayContainer>
+        <ContentBox>
           <h2>
             <Translation id="page-index-network-stats-title" />
           </h2>
-          <Subtitle>
+          <SectionDecription>
             <Translation id="page-index-network-stats-subtitle" />
-          </Subtitle>
-        </StyledContent>
+          </SectionDecription>
+        </ContentBox>
         <StatsBoxGrid />
-      </StyledGrayContainer>
-      <StyledContent>
+      </GrayContainer>
+      {/* Explore Section */}
+      <ContentBox>
         <h2>
           <Translation id="page-index-touts-header" />
         </h2>
```

Comment:
I think this entire `h2` should be inside the below `ContentBox` section. If not this gets center aligned (and it should be left aligned).
![image](https://user-images.githubusercontent.com/468158/218876161-c4477916-a798-4386-b9e7-036cfd4335a9.png)